repository/diet_repository.py

```diff
@@ -0,0 +1,57 @@
+from domain.entity.diet_requirement import DietRequirement
+
+
+def save_dietary_requirements(session, user_id, dietary_requirements):
+    """
+    A helper function for upload the data to the database.
+    This function will map the argument from the frontend with the statement in the database
+    Then using the session to update the database
+    This function is allowed to change the dietary requirement for the user by providing the user_id
```

Comment:
Good comment, but a little too much detail in this first part. Something like "Stores dietary requirements for the given user to the database. Any existing requirements for that user will be overwritten." These comments should focus on what the function does, rather than how it does it (we should be able to read the code to work out the how).